next/docs/iota-sdk/docs/how-tos/alias-wallet/request-funds.mdx

```diff
@@ -19,19 +19,170 @@ import TabItem from '@theme/TabItem';
 import TipFaucet from '../../_admonitions/_tip-faucet.md';
 import AliasNFT from '../../_admonitions/_alias_nft_wallet.md';
 
-Alias and NFT outputs have their own address and therefore can hold their own base token or any other native asset.
+Alias addresses play a significant role in decentralized systems, particularly in managing and identifying various assets.
```

Comment:
This paragraph talks about Alias addresses, but NFT addresses are not Alias addresses. The text should rather reflect that the Alias and NFT outputs have their own addressable address and hence can hold other assets.